code/category/test_script_for_concept_catgorization.py

```diff
@@ -0,0 +1,164 @@
+# from code.models import models
+import random
+from sklearn.cluster import KMeans
+from collections import Counter
+
+
+class concept_categorization:
+    def __init__(self):
+        self.member = []
+
+    def add_member(self, member_list):
+        self.member.append(member_list)
+
+    def get_score(self):
+        correct = 0
+        for i in range(len(self.member)):
+            if self.member[i][0]==self.member[i][2]:
+                correct+=1
+        return correct/len(self.member)
+
+class embedding_model:
```

Comment:
I think you don't need this class, simply pass the W2V class.  So your methods simply should accept a loaded model.